modules/planning/open_space/trajectory_smoother/distance_approach_ipopt_interface.cc

```diff
@@ -2288,23 +2289,30 @@ bool DistanceApproachIPOPTInterface::eval_h(int n, const double* x, bool new_x,
     // return the structure. This is a symmetric matrix, fill the lower left
     // triangle only.
 
-    for (int idx = 0; idx < nnz_L; idx++) {
-      iRow[idx] = rind_L[idx];
-      jCol[idx] = cind_L[idx];
+    if (!fill_lower_left(iRow, jCol, rind_L, cind_L, nnz_L)) {
+        // use CPU if Cuda initialize failed
+        for (int idx = 0; idx < nnz_L; idx++) {
+          iRow[idx] = rind_L[idx];
+          jCol[idx] = cind_L[idx];
+        }
     }
   } else {
     // return the values. This is a symmetric matrix, fill the lower left
     // triangle only
 
     obj_lam[0] = obj_factor;
-    for (int idx = 0; idx < m; idx++) obj_lam[1 + idx] = lambda[idx];
+    if (!data_transfer(&obj_lam[1], lambda, m)) {
+        for (int idx = 0; idx < m; idx++) obj_lam[1 + idx] = lambda[idx];
```

Comment:
Fix indents. Plz try "scripts/clang-format.sh" if you are not sure about styles.

Reply:
Sorry for my fault. I fix this issue ASAP